lib/contents.php

```diff
@@ -257,9 +243,6 @@ function getContents($url, $header = array(), $opts = array(), $returnHeader = f
 			$retVal['content'] = $cache->loadData();
 			break;
 		default:
-			if(array_key_exists('server', $finalHeader) && stripos($finalHeader['server'], 'cloudflare') !== false) {
-				throw new CloudflareChallengeException($errorCode);
```

Comment:
Removed is executed if response code is not 200, 201, 202.
What is wrong here?

Reply:
*removed code is executed

Reply:
There are lots of websites who respond with `cloudflare` in the `server` header. They are usually not the cloudflare captcha but it simply means they are using cloudflare as a frontend (CDN).